main.go

```diff
@@ -62,7 +63,7 @@ func main() {
 		}
 	}
 
-	appConfig, err := config.NewAppConfig("lazygit", version, commit, date, buildSource, debuggingFlag)
+	appConfig, err := config.NewAppConfig("lazygit", version, commit, date, buildSource, debuggingFlag, nil)
```

Comment:
I'm slightly confused here: we're allowing a services map to be passed in but I can't see anywhere that we are passing it in here. It seems to me that we don't need services mentioned in main.go and can leave it to app_config.go

Reply:
heh true, seems I made some over-engineer there 😅